tests/unit/lms/services/lti_grading/_v11_test.py

```diff
@@ -135,39 +211,7 @@ def test_requests_fail_if_body_not_xml(self, svc, http_service):
             ExternalRequestError,
             match="Unable to parse XML response from LTI Outcomes service",
         ):
-            svc.read_result(self.GRADING_ID)
-
-    def test_requests_fail_if_no_status(self, svc, respond_with):
-        respond_with(include_status=False)
-        with pytest.raises(
-            ExternalRequestError, match="Malformed LTI outcome response"
-        ):
-            svc.read_result(self.GRADING_ID)
-
-    def test_requests_fail_if_response_is_malformed(self, svc, respond_with):
-        respond_with(malformed=True)
-
-        with pytest.raises(
-            ExternalRequestError, match="Malformed LTI outcome response"
-        ):
-            svc.read_result(self.GRADING_ID)
-
-    def test_requests_fail_if_status_is_not_success(self, svc, respond_with):
-        respond_with(status_code="failure")
-
-        # LTI outcome request failed
-        with pytest.raises(
-            ExternalRequestError,
-            match="<imsx_description>An error occurred.</imsx_description>",
-        ):
-            svc.read_result(self.GRADING_ID)
-
-    def test_requests_fail_and_no_description_returned(self, svc, respond_with):
-        respond_with(status_code="failure", include_description=False)
-
-        # imsx_description is missing
-        with pytest.raises(ExternalRequestError, match="LTI outcome request failed"):
-            svc.read_result(self.GRADING_ID)
```

Comment:
All of these are actually `read_result` tests, but for some reason aren't named like that. They also are positioned after the tests for the other method. I've renamed these and put them to the top.